enatega-multivendor-admin/src/components/Restaurant/CreateRestaurant.jsx

```diff
@@ -66,10 +69,14 @@ const CreateRestaurant = props => {
   const [minimumOrderError, setMinimumOrderError] = useState(null)
   const [salesTaxError, setSalesTaxError] = useState(null)
   const [restaurantCuisines, setRestaurantCuisines] = React.useState([])
-
   // const [shopType, setShopType] = useState(SHOP_TYPE.RESTAURANT)
   const [errors, setErrors] = useState('')
   const [success, setSuccess] = useState('')
+  const [open, setOpen] = React.useState(true)
+
+  const handleOpen = () => setOpen(true);
```

Comment:
This function is not called anywhere in this component, you are setting state to false on close click, but what if user again wants to open a modal, will click from parent component opens the modal ? as local state of this component is already false. 
In my previous change, I requested to update the parent state directly instead of creating new local state.